src/pages/Host/LiveSession/HostLiveSession.css

```
@@ -1,1107 +1,1219 @@
+
+.host-live-wave-bar:nth-child(4) {
+  animation-delay: 0.1s;
+  height: 30px;
+}
+
+.host-live-wave-bar:nth-child(5) {
+  animation-delay: 0.25s;
+  height: 14px;
+}
+
+@keyframes waveBounce {
```

Stylelint (17.7.0)

[error] Expected keyframe name "waveBounce" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)